installation-bundle/src/InstallTool.php

```diff
@@ -151,14 +151,7 @@ public function isFreshInstallation(): bool
             return true;
         }
 
-        $statement = $this->connection->query('
-            SELECT
-                COUNT(*) AS count
-            FROM
-                tl_page
-        ');
-
-        return $statement->fetch(\PDO::FETCH_OBJ)->count < 1;
+        return $this->connection->fetchOne('SELECT COUNT(*) AS count FROM tl_page') < 1;
```

Comment:
```suggestion
        return $this->connection->fetchOne('SELECT COUNT(*) FROM tl_page') < 1;
```

Reply:
Changed in efd72b05bed5fb7afde25e4688f9d878c79bf440.